invisible_cities/cities/base_cities.py

```diff
@@ -756,8 +756,8 @@ def compute_xy_position(self, s2si, peak_no):
         """
         IDs, Qs = cpmp.integrate_sipm_charges_in_peak(s2si, peak_no)
         xs, ys   = self.xs[IDs], self.ys[IDs]
-        #return self.reco_algorithm(np.stack((xs, ys)).T, Qs)
-        return corona(np.stack((xs, ys)).T, Qs,
+        #return self.reco_algorithm(np.stack((xs, ys), axis=1), Qs)
```

Comment:
Is there any good reason for keeping this commented-out line?

Reply:
There is a similar one in the `__init__` method than can be removed as well.